### DISCUSSION TITLE
Installed AdGuard on Mac mini - But clients sometimes seem to take a while to get online?

### DISCUSSION THREAD

**Discussion by brettstark73** [AdguardTeam/AdGuardHome#7487] [Q&A]
Hello - so, a lot of time, everything works as normal/fine. At times though, clients ie phones or TVs seem to not be online. They can take quite some time to get connected to the internet. Is this because the Mac mini was asleep and is waking up or something? That is all I can think of. In Mac M1 settings, I have prevented auto sleeping, and now tried disabling 'put hard disks to sleep when possible. I have 'wake for network access' on. Anything else to do or check please on Mac or AdGuard? Everything in my house is wireless including the mac mini via a TP Link mesh network.

**Comment by brettstark73** [comment]
Any answer to this thanks? Problem is still happening. I will open an issue.

**Comment by Dynamic5912** [comment]
Duplicated here https://github.com/AdguardTeam/AdGuardHome/issues/7564
